scipy/stats/_sampling.py

```diff
@@ -1140,15 +1140,11 @@ class RatioUniforms:
         The upper bound of the bounding rectangle in the v-direction.
     c : float, optional.
         Shift parameter of ratio-of-uniforms method, see Notes. Default is 0.
-    random_state : {None, int, `numpy.random.Generator`,
-                    `numpy.random.RandomState`}, optional
-
-        If `seed` is None (or `np.random`), the `numpy.random.RandomState`
-        singleton is used.
-        If `seed` is an int, a new ``RandomState`` instance is used,
-        seeded with `seed`.
-        If `seed` is already a ``Generator`` or ``RandomState`` instance then
-        that instance is used.
+    rng : {None, int, `numpy.random.Generator`}, optional
+        If `rng` is an int or None, a new `numpy.random.Generator` is
+        created using ``np.random.default_rng(rng)``.
+        If `rng` is already a ``Generator`` instance, then the provided
+        instance is used.
```

Comment:
```suggestion
        instance is used.

        .. versionchanged::
            Before SciPy 1.15, this parameter was called ``random_state`.
            For backwards compatibility this will remain valid for the time being.
```
As Ralf noted above, we should mention the old name somewhere, a ``versionchanged`` directive might be a discoverable enough and serve the double purpose of informing about changes nicely.